src/transformers/models/swiftformer/modeling_tf_swiftformer.py

```diff
@@ -0,0 +1,911 @@
+        )  # FIXME
+        self.point_wise_conv1 = tf.keras.layers.Conv2D(hidden_dim, kernel_size=1, name="point_wise_conv1")
+        self.act = get_tf_activation("gelu")
+        self.point_wise_conv2 = tf.keras.layers.Conv2D(dim, kernel_size=1, name="point_wise_conv2")
+        self.drop_path = tf.keras.layers.Identity(name="drop_path")  # FIXME: is this supposed to be like this?
+        self.hidden_dim = int(config.mlp_ratio * self.dim)
+
+    def build(self, input_shape=None):
+        self.layer_scale = self.add_weight(
+            name="layer_scale",
+            shape=(self.dim),  # TODO: check this
+            initializer="ones",
+            trainable=True,
+        )
+
+        if self.built:
+            return
+        self.built = True
+        if getattr(self, "depth_wise_conv", None) is not None:
+            with tf.name_scope(self.depth_wise_conv.name):
+                self.depth_wise_conv.build(self.dim)
+        if getattr(self, "norm", None) is not None:
+            with tf.name_scope(self.norm.name):
+                self.norm.build((None, None, None, self.dim))
+        if getattr(self, "point_wise_conv1", None) is not None:
+            with tf.name_scope(self.point_wise_conv1.name):
+                self.point_wise_conv1.build(self.dim)
+        if getattr(self, "point_wise_conv2", None) is not None:
+            with tf.name_scope(self.point_wise_conv2.name):
+                self.point_wise_conv2.build(self.hidden_dim)
+        if getattr(self, "drop_path", None) is not None:
+            with tf.name_scope(self.drop_path.name):
+                self.drop_path.build(None)
+
+    def call(self, x: tf.Tensor, training: bool = False) -> tf.Tensor:
+        input = x
+        x = self.pad(x)
+        x = self.depth_wise_conv(x)
+        x = self.norm(x, training=training)
+        x = self.point_wise_conv1(x)
+        x = self.act(x)
+        x = self.point_wise_conv2(x)
+        x = input + self.drop_path(self.layer_scale * x)
+        return x
+
+
+class TFSwiftFormerMlp(tf.keras.layers.Layer):
+    """
+    MLP layer with 1*1 convolutions.
+
+    Input: tensor of shape `[batch_size, channels, height, width]`
+
+    Output: tensor of shape `[batch_size, channels, height, width]`
+    """
+
+    def __init__(self, config: SwiftFormerConfig, in_features: int, **kwargs):
+        super().__init__(**kwargs)
+
+        hidden_features = int(in_features * config.mlp_ratio)
+        self.norm1 = tf.keras.layers.BatchNormalization(
+            epsilon=config.batch_norm_eps, momentum=0.9, name="norm1"
+        )  # FIXME: is this the correct momentum?
+        self.fc1 = tf.keras.layers.Conv2D(hidden_features, 1, name="fc1")
+        act_layer = get_tf_activation(config.hidden_act)
+        self.act = act_layer
+        self.fc2 = tf.keras.layers.Conv2D(in_features, 1, name="fc2")
+        self.drop = tf.keras.layers.Dropout(rate=0.0)  # FIXME: is this supposed to be 0?
```

Comment:
I doubt it - it would expect this to be controlled by the config

Reply:
Addressed here https://github.com/huggingface/transformers/pull/23342/commits/3cf26e610e04e4de7802a7dc3fdc27ace58c49e8

Reply:
Readdressed here: https://github.com/huggingface/transformers/pull/23342/commits/aa7cb413df01245c6787d77d13eb4b46030c9c70